auth/transports/simplified_http_transport.go

```diff
@@ -0,0 +1,112 @@
+package transports
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sync"
+
+	"github.com/bsv-blockchain/go-sdk/auth"
+)
+
+// SimplifiedHTTPTransport implements the Transport interface for HTTP communication
+type SimplifiedHTTPTransport struct {
+	url         string
+	client      *http.Client
+	onDataFuncs []func(*auth.AuthMessage) error
+	mu          sync.Mutex
+}
+
+// SimplifiedHTTPTransportOptions represents configuration options for the transport
+type SimplifiedHTTPTransportOptions struct {
+	URL         string
+	PollEnabled bool
+	PollTimeout int
+}
+
+// NewSimplifiedHTTPTransport creates a new HTTP transport instance
+func NewSimplifiedHTTPTransport(options *SimplifiedHTTPTransportOptions) (*SimplifiedHTTPTransport, error) {
+	if options.URL == "" {
+		return nil, errors.New("URL is required for HTTP transport")
+	}
+
+	return &SimplifiedHTTPTransport{
+		url:    options.URL,
+		client: &http.Client{},
+	}, nil
+}
+
+// Send sends an AuthMessage via HTTP
+func (t *SimplifiedHTTPTransport) Send(message *auth.AuthMessage) error {
+	jsonData, err := json.Marshal(message)
+	if err != nil {
+		return fmt.Errorf("failed to marshal auth message: %w", err)
+	}
+
+	resp, err := t.client.Post(t.url, "application/json", bytes.NewBuffer(jsonData))
+	if err != nil {
+		return fmt.Errorf("failed to send HTTP request: %w", err)
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
+		body, _ := ioutil.ReadAll(resp.Body)
+		return fmt.Errorf("HTTP request failed with status %d: %s", resp.StatusCode, string(body))
+	}
+
+	// If we have a response, process it as a potential auth message
+	if resp.ContentLength > 0 {
+		var responseMsg auth.AuthMessage
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return fmt.Errorf("failed to read response body: %w", err)
+		}
+
+		err = json.Unmarshal(body, &responseMsg)
+		if err != nil {
+			// Not a valid auth message, just ignore
```

Comment:
Hmmm, maybe this is kinda general thought. But I think some logging tool would be beneficial here (and for such situations).

BTW: We picked `log/slog` because it is "standard" and has many adapters for "almost" every logger (like zerolog etc...)

```suggestion
			slog.Debug("Received non-auth message", "body", string(body)) //it uses default logger
			
			// or if logger is passed as SimplifiedHTTPTransportOptions for example
			logger.Debug("Received non-auth message", "body", string(body)) //it uses default logger
			
```